fiasco/tests/test_ion.py

```diff
@@ -346,52 +339,42 @@ def test_free_free(ion):
     # This value has not been tested for correctness
     assert u.allclose(emission[0], 1.72804216e-29 * u.cm**3 * u.erg / u.Angstrom / u.s)
 
-def test_gaunt_factor_free_free_total(ion):
-    gf = ion._gaunt_factor_free_free_total()
-    assert gf.shape == ion.temperature.shape
-    # This value has not been tested for correctness
-    assert u.allclose(gf[0], 1.23584439 * u.dimensionless_unscaled)
-
-def test_free_free_radiative_loss(h1, fe20):
+@pytest.mark.parametrize(('index','expected'),
+                        [(0, 1.79093013e-22),
+                        (75, 3.06577525e-21), ])
+def test_free_free_radiative_loss(h1, fe20, index, expected):
     assert fe20.free_free_radiative_loss().shape == fe20.temperature.shape
     assert u.allclose(h1.free_free_radiative_loss(), 0.0 * u.erg * u.cm**3 / u.s)
     # This value has not been tested for correctness
-    assert u.isclose(fe20.free_free_radiative_loss()[0], 1.79093013e-22 * u.erg * u.cm**3 / u.s)
+    assert u.isclose(fe20.free_free_radiative_loss()[index], expected * u.erg * u.cm**3 / u.s)
+
+@pytest.mark.requires_dbase_version('>= 9.0.1')
+@pytest.mark.parametrize(('index','expected'),
+                        [(0, 1.79093013e-22),
+                        (75, 3.05628894e-21), ])
+def test_free_free_radiative_loss_itoh(h1, fe20, index, expected):
+    """
+    For database versions >= 9.0.1, the Itoh Gaunt factors give a different
+    result for the free-free radiative loss at temperature > 1 MK.
+    """
+    assert fe20.free_free_radiative_loss(use_itoh=True).shape == fe20.temperature.shape
+    assert u.allclose(h1.free_free_radiative_loss(use_itoh=True), 0.0 * u.erg * u.cm**3 / u.s)
+    # This value has not been tested for correctness
+    assert u.isclose(fe20.free_free_radiative_loss(use_itoh=True)[index], expected * u.erg * u.cm**3 / u.s)
```

Comment:
Not sure about that.  The value returned by `fe20.free_free_radiative_loss(use_itoh=True)[75]` depends on the database version.  For database versions < 9.0.1, it will give the Sutherland result, and for > 9.0.1 it will give the Itoh result, which is slightly different.

Reply:
In other words, if I just parametrized with `use_itoh`, the pytest will fail for either the newer or older versions.  I would need to parametrize on the database version -- is that possible?

Reply:
Oops I missed the `requires_dbase_version` mark on the second test. There might be a way to combine them with that mark somehow, but it would probably be overly convoluted. Leaving them separate is fine. Sorry for the confusion!